src/components/Tile/Tile.js

```diff
@@ -416,6 +422,9 @@ export class ExpandableTile extends Component {
     return React.Children.map(this.props.children, child => child);
   };
 
+  // a unique ID generated for use in aria-labelledby if one isn't providedj
+  uid = uid();
```

Comment:
originally I meant instead of creating a new variable wouldn't we just be able to do this?

```suggestion
  uid = uid();
  uid = this.props.id || uid();
```

Reply:
@emyarod Doing so prevents change in `props.id` after initialization being ignored.